addons/l10n_hu_edi/models/l10n_hu_edi_credentials.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError
+from odoo.addons.l10n_hu_edi.models.l10n_hu_edi_connection import L10nHuEdiConnectionError
+
+
+_PROTECTED_FIELDS = {'mode', 'vat', 'username', 'password', 'signature_key', 'replacement_key'}
+
+
+class L10nHuEdiCredentials(models.Model):
+    _name = 'l10n_hu_edi.credentials'
+    _description = 'Hungarian TAX Authority Login Credentials'
+    _rec_name = 'username'
+    _order = 'is_active desc'
+
+    # === User interface fields === #
+    is_active = fields.Boolean(
+        string='Active',
+        default=True,
+    )
+    is_primary = fields.Boolean(
+        string='Is Primary',
+        compute='_compute_is_primary',
+    )
+
+    # === Data fields === #
+    mode = fields.Selection(
```

Comment:
We could just have them on the company?

Reply:
This might be a drawback for support, because the test credentials are different from production credentials. So, if we only let users put in one set of credentials at a time, they will put their production credentials, and tech support will not be able to switch to test credentials to debug issues.